packages/myst-cli/src/utils/addWarningForFile.ts

```diff
@@ -26,7 +27,18 @@ export function addWarningForFile(
   const url = opts?.url ? chalk.reset.dim(`\n   See also: ${opts.url}\n`) : '';
   const prefix = file ? `${file}${line}${column} ` : '';
   const formatted = `${message}${note}${url}`;
-  switch (kind) {
+  if (opts?.ruleId) {
+    const config = selectCurrentProjectConfig(session.store.getState());
+    const handler = config?.error_rules?.find((rule) => rule.id === opts.ruleId);
+    if (handler) {
+      if (handler.severity === 'ignore') {
+        session.log.debug(`${prefix}${formatted}`);
+        return;
+      }
+      severity = handler.severity as WarningKind;
+    }
+  }
+  switch (severity) {
```

Comment:
After this switch statement, we need to add
```typescript
session.log.debug(`To suppress this message, add "${rule.id}" to "error_rules" in your project config`)
```

Reply:
This make me think about my previous comment about default severity. It's easier to say "add X to error_rules" than to say "add an entry to error_rules with id X and severity info" - so maybe that means the default is `severity: info`. In that case, maybe it should be, like `error_exceptions`?

```yaml
error_exceptions:
  - my-id
```

sounds like we are excepting that error. But

```yaml
error_exceptions:
  - id: my-id
    severity: warn
```

also still makes sense - it's an exception because we are changing the severity.

Reply:
Also... maybe we don't want to tell the user how to suppress the message. That might make it too easy to abuse...? _But_ we definitely need to debug log the `ruleId`; that way, if they read the docs, they have all the info they need to suppress the message.

If we don't log the `ruleId` this whole feature feels inaccessible to users.